### PR TITLE
Warn if using earth instead of earthly

### PR DESCRIPTION
**Pull request by alexcb**
Signed-off-by: Alex Couture-Beil <alex@earthly.dev>

### CODE REVIEW

File: cmd/earthly/main.go [hunk 1]
```diff
@@ -792,7 +792,21 @@ func (app *earthlyApp) before(context *cli.Context) error {
...
 
+func (app *earthlyApp) warnIfEarth() {
+	if len(os.Args) == 0 {
+		return
+	}
+	binPath := os.Args[0] // can't use os.Executable() here; because it will give us earthly if executed via the earth symlink
+
+	baseName := path.Base(binPath)
+	if baseName == "earth" {
+		app.console.Warnf("Warning: the earth binary has been renamed to earthly; the earth command is currently symlinked, but is deprecated and will one day be removed.")
```
Comment (vladaionescu) [inline cmd/earthly/main.go:803]: We should also say... `Feel free to rm <path-to-earth>`, if we detect that `earthly` exists.

**Review by vladaionescu — APPROVED** [review]

File: cmd/earthly/main.go (hunk 1 above)
Comment (alexcb) [reply cmd/earthly/main.go:803]: that's a good idea, I just pushed up a new version to do that.

Comment (vladaionescu) [reply cmd/earthly/main.go:803]: Forgot to add files maybe?

Comment (alexcb) [reply cmd/earthly/main.go:803]: I don't think so, I did a rebase+force push.

Comment (vladaionescu) [reply cmd/earthly/main.go:803]: Oh I see https://github.com/earthly/earthly/pull/637

Comment (vladaionescu) [reply cmd/earthly/main.go:803]: nvm, I got wires crossed :-)

**Review by vladaionescu — APPROVED** [review]

**Comment by vladaionescu** [comment]
I think there is an issue:

```
~/workspace/earthly main ❯ ./build/linux/amd64/earthly --ci --push -P ./examples/cpp+docker
Once you are ready to switch over to earthly, you can `rm /home/vlad/workspace/earthly/build/linux/amd64/earthly`
```

(I ran earthly, not earth)

**Comment by alexcb** [comment]
Thanks for that fix, good catch.

**Pull request merged**